modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/theme-portlet-settings.json

```
@@ -5,7 +5,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
+				"FileName": "minium_site_navigation.ftl",
```

Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
```